src/scenes/home/softSkillsList.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { SoftSkillsItem } from "./softSkillsItem";
+
+export function SoftSkillsList() {
+  const softSkills = ["patient", "ambitious", "fast learner"];
```

Comment:
i would move softskilss outside of the component, and replace  createList function with inline code

Reply:
Done.